linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/istio/HttpIstioRequest.scala

```diff
@@ -0,0 +1,11 @@
+package io.buoyant.linkerd.protocol.http.istio
+
+import com.twitter.finagle.http.Request
+import io.buoyant.k8s.istio.{CurrentIstioPath, IstioRequest}
+
+object HttpIstioRequest {
+  def apply(req: Request): IstioRequest[Request] =
```

Comment:
I think we should pass `Option[IstioPath]` in here so that callers are explicit about when they are pulling the local context.

Reply:
This means that code in the identifiers that construct an HttpIstioRequest can explicitly pass None for IstioPath